docs/src/reference/workflow-config-file.rst

```diff
@@ -936,6 +936,33 @@ no_timetree
 -  description: Do not produce a time tree.
 -  default: ``false``
 
+colors
+------
+
+-  type: object
+-  description: Parameters for assigning colors in ``scripts/assign-colors.py``
+-  examples:
+
+.. code:: yaml
+
+   colors:
+     default:
+       clade_recency: ""
```

Comment:
I think there is a mismatch between these docs and the code implementation. In the workflow code, the `_get_clade_recency_for_wildcards` function looks for `config["colors"]["clade_recency"]` as the default value instead of a `default` key.

I slightly prefer the `default` key over `clade_recency`.

Reply:
In 82ca8d48f671349c5688a055bcc67e108c153c54 I've switched behavior to use `config["colors"]["default"]["clade_recency"]` when looking for a default value. @huddlej: Your preference of `default` over `clade_recency` matches @joverlee521, @victorlin and @jameshadfield's preferences expressed in https://github.com/nextstrain/ncov/issues/1131.